src/components/Signup/PodRegistrationForm.jsx

```diff
@@ -0,0 +1,152 @@
+// React Imports
+import React, { useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+// MUI imports
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+import { Visibility, VisibilityOff } from '@mui/icons-material';
+import FilledInput from '@mui/material/FilledInput';
+import IconButton from '@mui/material/IconButton';
+import InputAdornment from '@mui/material/InputAdornment';
+import InputLabel from '@mui/material/InputLabel';
+import Button from '@mui/material/Button';
+import Card from '@mui/material/Card';
+import CardHeader from '@mui/material/CardHeader';
+
+/* Styles for MUI components */
+const cardStyle = {
+  padding: '8px',
+  margin: '8px',
+  boxShadow: '0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19)'
+};
+
+const formRowStyle = {
+  margin: '20px 0'
+};
+
+const textFieldStyle = {
```

Comment:
`TextField` has the option to use `margin="normal"`, which adds the standard amount of margin. This is what our other text fields currently use.